plugins/callback/foreman.py

```diff
@@ -132,8 +132,10 @@ def get_time():
 def get_now():
     """
     Return the current timestamp as a string to be sent over the network.
+    The time is always in UTC timezone and printed in a format Ruby
+    DateTime can easily parse.
     """
-    return datetime.utcnow().isoformat()
+    return datetime.datetime.now(datetime.timezone.utc).isoformat(sep=" ", timespec="seconds")
```

Comment:
```
>>> datetime.datetime.now(datetime.timezone.utc).isoformat(sep=" ", timespec="seconds")
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    datetime.datetime.now(datetime.timezone.utc).isoformat(sep=" ", timespec="seconds")
AttributeError: type object 'datetime.datetime' has no attribute 'datetime'
```

The `datetime` here is `datetime.datetime` already. You want to change the import if you need both `datetime.datetime` and `datetime.timezone`.

Reply:
The correct, version agnostic way to get the UTC time is `datetime.datetime.utcnow()`

Now I just need to understand why you set `sep` and `timespec` (as the later is not understood by Python 2)